hapi/src/workers/producers.worker.js

```diff
@@ -47,17 +47,17 @@ const start = async () => {
     producerService.syncProducers,
     workersConfig.syncProducersInterval
   )
-  run(
-    'SYNC PRODUCER INFO',
-    producerService.syncProducersInfo,
-    workersConfig.syncProducerInfoInterval
-  )
-  run(
-    'SYNC CPU USAGE',
-    producerService.syncCpuUsage,
-    workersConfig.syncProducerCpuInterval
-  )
-  run('CHECK FOR MISSED BLOCK', producerService.checkForMissedBlocks)
+  // run(
```

Comment:
@adriexnet why is this commented?

Reply:
My mistake, it was fixed in the las commit